nss-day-cohort-27/nss-tribute-agency-the-towering-crabs
# Executive summary

To test
checkout to master  branch
git fetch --all
git checkout executive-summary

run http-server

page should show portrait and info
